src/qibocal/protocols/randomized_benchmarking/utils.py

```diff
@@ -572,7 +572,7 @@ def layer_circuit(rb_gen: Callable, depth: int, target) -> tuple[Circuit, dict]:
     """
     full_circuit = None
     random_indexes = []
-    if isinstance(target, int):  # int for qubit
+    if isinstance(target, QubitId):
```

Comment:
```suggestion
    if isinstance(target, (str,int)):
```
This should fix tests.